src/sagemaker_containers/env.py

```diff
@@ -470,10 +475,19 @@ def hosts(self):  # type: () -> list
         """The list of names of all containers on the container network, sorted lexicographically.
                 For example, `["algo-1", "algo-2", "algo-3"]` for a three-node cluster.
         Returns:
-              list[str]: all the hosts in the training network.
+              (list[str]): all the hosts in the training network.
```

Comment:
(\<type\>) is not google doc style for return values (even though it is for args)